accepted/2020/workloads/workload-manifest.md

```diff
@@ -122,9 +122,9 @@ Manifests may also provide logic to import workload packs automatically via a `W
 
 ## Packaging
 
-Manifests are packaged and distributed as NuGet packages. They will use the existing package type called `DotnetPlatform` so they cannot be accidentally used as a `PackageReference`. The manifest’s `sdk-manifest.json` and `WorkloadManifest.targets` are in the root folder of the NuGet. The ID of the NuGet is `{manifest-id}-{sdk-band}` and the version of the NuGet is the manifest version.
+Manifests are packaged and distributed as NuGet packages. The ID of the NuGet package is `{manifest-id}.Manifest-{sdk-band}` (for example, `Microsoft.NET.Sdk.Android.Manifest-6.0.200`) and the version of the NuGet package is the manifest version. They will use the existing package type called `DotnetPlatform` so they cannot be accidentally used as a `PackageReference`. The manifest’s `workloadmanifest.json`, `WorkloadManifest.targets`, and any other files are in the `data/` folder of the NuGet package. This ensures that there's not confusion about whether a file in the root is part of the workload manifest or a file owned by NuGet. 
```

Comment:
is the proper name `Microsoft.NET.Sdk.Android.Manifest-6.0.200` or `Microsoft.NET.Workload.Android.Manifest-6.0.200` since the Manifest id is `Microsoft.NET.Workload.Android`

Reply:
If the manifest ID is `Microsoft.NET.Workload.Android` then the NuGet package ID for the manifest would be `Microsoft.NET.Workload.Android.Manifest-6.0.200`.

Reply:
Understood, I was just pointing out that we do expect the {manifest-id} and the package name to align and it seems like we're going for Microsoft.NET.Workload.<name> for the manifests so it would be good to correct the example